social-network/src/PromoBox/PromoBox.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './PromoBox.css';
+import CitationBox from './CitationBox';
+import MemberPreviewIcon from './MemberPreviewIcon';
+import AudioElement from './AudioElement';
+
+class PromoBox extends Component {
+
+  getMemberPreview() {
+    return (
+      <div className="member-preview">
+        <MemberPreviewIcon />
+        <span>Member preview</span>
+      </div>
+    );
+  }
+
+  getAudioElement() {
+    return (
+      <AudioElement />
+    );
+  }
+
+  getAudioMemberElement() {
+    return (
+      <div className="audio-and-member">
+        <MemberPreviewIcon />
+        <AudioElement />
+      </div>
+    );
+  }
+
+
+  render() {
+    const {
+      title, description, image, link, author, postedDate, minutesToRead, hasAudioAvailable, memberPreview
+    } = this.props.promoInfo;
```

Comment:
Some of these variables are unused. You may want to remove the ones that aren't being used. For props like `audioIcon`, you can do something like this:

```js
const { hasAudioAvailable } = this.props.promoInfo;
```

or

```js
const { hasAudioAvailable } = this.props.promoInfo;
const audioIcon = hasAudioAvailable;
```

or

```js
const { hasAudioAvailable: audioIcon } = this.props.promoInfo;
```

As far as the right way... it's just personal preference.